tests/Sitemap/SitemapPageRepositoryTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Tests\Sitemap;
+
+use Doctrine\Website\Sitemap\SitemapPageRepository;
+use PHPUnit\Framework\TestCase;
+use function date;
+
+class SitemapPageRepositoryTest extends TestCase
```

Comment:
[Scrutinizer](https://scrutinizer-ci.com/g/doctrine/doctrine-website/inspections/85686785-d93a-4eab-8364-25a203c01ff7/code-structure/class/Doctrine%5CWebsite%5CSitemap%5CSitemapPageRepository?expandCoverage=1) is showing uncovered code. We need to cover that code or remove it, in case it is unreachable.

Reply:
I added those checks because phpstan complains otherwise. It never happens in the real scenario.

Reply:
I am making some changes so it can be removed.